opcua/binary_client.py

```diff
@@ -10,21 +10,12 @@
 import opcua.uaprotocol as ua
 import opcua.utils as utils
 
-
-class CachedRequest(object):
```

Comment:
Is this normal? It look slike you reverted some stuff I merged this morning.....